pillar/cni.sls

```diff
@@ -20,7 +20,5 @@ flannel:
 cni:
   plugin:         'flannel'
   dirs:
-    #bin:          '/opt/cni/bin'
-    # TODO: use the standard directory
```

Comment:
The idea of using the standard directory is valid, So I would leave the comment or at least say why we are not using the standard directory...

Reply:
Good point, updating.